corpora/proverbs.dep

```diff
@@ -0,0 +1,1826 @@
+"<ташын>"
+	"таш" n px3sp acc @obj #3->4
+"<алат>"
+	"ал" v tv aor p3 sg @root #4->0
+"<,>"
+	"," cm @punct #5->9
+"<кыш>"
+	"кыш" n nom @obj #6->7
+"<койгон>"
+	"кой" v tv gpr_past subst nom @nsubj #7->9
+"<кышын>"
+	"кыш" n px3sp acc @obj #8->9
+"<алат>"
+	"ал" v tv aor p3 sg @parataxis #9->4
+"<.>"
+	"." sent @punct #10->9
+
+# sent_id = proverbs:69
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:235
+# text = Жыгылсаң нардан жыгыл, буйласын кармай жыгыл.
+# text[eng] = If you fall, fall off a (dromedary) camel, fall holding its nose peg.
+# labels = JNW_checked_cg tocheck_dep
+"<Жыгылсаң>"
+	"жык" v tv pass prc_cond p2 sg @advcl #1->3
+"<нардан>"
+	"нар" n abl @obl #2->3
+"<жыгыл>"
+	"жык" v tv pass imp p2 sg @root #3->0
+"<,>"
+	"," cm @punct #4->7
+"<буйласын>"
+	"буйла" n px3sp acc @obj #5->6
+"<кармай>"
+	"карма" v tv prc_impf @advcl #6->7
+"<жыгыл>"
+	"жык" v tv pass imp p2 sg @parataxis #7->3
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:70
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:21
+# text = Ак ийилет, сынбайт.
+# text[eng] = Fairness/justice bends, [but] does not break.
+# labels = JNW_checked_cg tocheck_dep
+"<Ак>"
+	"ак" n nom @nsubj #1->2
+"<ийилет>"
+	"ий" v tv pass aor p3 sg @root #2->0
+"<,>"
+	"," cm @punct #3->4
+"<сынбайт>"
+	"сын" v iv neg aor p3 sg @parataxis #4->2
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:71
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:479
+# text = Энеси болушчаактын кызы ыйлаак, атаcы болушчаактын уулу урушчаак.
+# text[eng] = The daughter of a protective mother cries a lot, the son of a protective father gets in fights a lot.
+# labels = JNW_checked_cg tocheck_dep
+"<Энеси>"
+	"эне" n px3sp nom @nmod:poss #1->3
+"<болушчаактын>"
+	"болушчаак" adj subst gen @nmod:poss #2->1
+"<кызы>"
+	"кыз" n px3sp @nsubj #3->4
+"<ыйлаак>"
+	"ыйлаак" adj @root #4->0
+		"э" cop aor p3 sg @cop #5->4
+"<,>"
+	"," cm @punct #6->10
+"<атаcы>"
+	"ата" n px3sp nom @nmod:poss #7->9
+"<болушчаактын>"
+	"болушчаак" adj subst gen @nmod:poss #8->7
+"<уулу>"
+	"уул" n px3sp nom @nsubj #9->10
+"<урушаак>"
+	"урушаак" adj @parataxis #10->4
+		"э" cop aor p3 sg @cop #11->10
+"<.>"
+	"." sent @punct #12->10
+
+# sent_id = proverbs:72
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:482
+# text = Эрди жоо үстүндө көр.
+# text[eng] = Know a man [by what he does] on the battle field.
+# labels = JNW_checked_cg tocheck_dep
+"<Эрди>"
+	"эр" n acc @obj #1->4
+"<жоо>"
+	"жоо" n nom @nsubj #2->4
+"<үстүндө>"
+	"үст" n px3sp loc @obl:lmod #3->1
+"<көр>"
+	"көр" v tv imp p2 sg @root #4->0
+"<.>"
+	"." sent @punct #5->4
+
+# sent_id = proverbs:73
+# author = Сыргабековна&Сабыровна
+# source = Сыргабековна&Сабыровна:401
+# text = Сүйбөгөнгө сүйкөнбө.
+# text[eng] = Don't go after someone who doesn't love you.
+# labels = JNW_checked_cg tocheck_dep
+"<Сүйбөгөнгө>"
+	"сүй" v tv neg gpr_past subst dat @obl #1->2
```

Comment:
`@advcl` is probably right, unless this is `@xcomp`?

Reply:
Nope, 2-years-ago-me, `@obl` seems right to now-me.  If the unexpressed noun were present, e.g. "сүйбөгөн кишиге сүйкөнбө", then it would almost certainly be `obl`.